cupy/logic/truth.py

```diff
@@ -57,3 +57,103 @@ def any(a, axis=None, out=None, keepdims=False):
 
     assert isinstance(a, cupy.ndarray)
     return a.any(axis=axis, out=out, keepdims=keepdims)
+
+
+def in1d(ar1, ar2, assume_unique=False, invert=False):
+    """
+    Test whether each element of a 1-D array is also present in a second array.
+
+    Returns a boolean array the same length as ``ar1`` that is True
```

Comment:
```suggestion
    Returns a boolean array the same length as ``ar1`` that is ``True``
```

Please do the same fix to all the similar instances.

Reply:
Fixed